packages/react/src/components/Forms/Select/mono-searchable.tsx

```diff
@@ -93,6 +93,10 @@ export const SelectMonoSearchable = forwardRef<SelectHandle, SubProps>(
       },
     }));
 
+    useEffect(() => {
+      props.onSearchInputChange?.({ target: { value: inputFilter } });
```

Comment:
I wonder if this is relevant to mimic the shape of an Event ? 🤔

Reply:
As the search input is also a regular input I thought it would be a better idea to use the same data type as the `onChange` callback of such inputs, WDYT?